dev.to/Slack-boost-your-teams-productivity.md

```diff
@@ -0,0 +1,107 @@
+![Alt Text](https://dev-to-uploads.s3.amazonaws.com/uploads/articles/hkxvg3vfdz4r4pridozn.png)
+
+##One Single Message
+There are a couple advantages on writing a single message:
+- Less notifications to your colleagues, not much to explain here, one message is equal to one notification, information spread on 4 separate messages, 4 notifications for each one on the channel.
+- Also talking about notifications, if the whole information is reduced to a single message, your partners can read it from the notification pop-up. If you split the information, you are forcing them to open the chat to know what you are talking about.
+- Organization and readability: if you write down and send messages as you are thinking of an idea, you are very likely to forget something, or maybe that information makes nonsense all spread out. If you take some more seconds and read the message before sending it, it’ll guarantee you that you’re not sending loose thoughts.
+- If you send several messages to express an idea, where should your teammates start the thread in order not to lose relevant information. It might not seem that serious, but imagine someone tags you on a thread created three days ago and you only have partial information on the initial message, you’ll be forced to scroll the chat to find some messages from a couple days ago. Pretty annoying, isn’t it?
+
+##Avoid Direct Messages when talking about the project
+This is a very common practice (or at least it was in the companies I worked on so far), to start a DM conversation between a few members of the team, or just to DM a partner to talk about something you don’t understand about the project or to make a decision.
+Avoiding this practice by starting a conversation on the team channel (you can also tag the people you consider relevant for that matter) bring several advantages:
+- The team knows what you are dealing with / working on.
+- Maybe people that you thought were not relevant can help you, or to bring an interesting solution to the problem, give their opinion, etc. (You are a team! This is a Key aspect)
+- Maybe you are working on something that overlaps with a task that a teammate is working on and you didn’t know.
+- Even if you are talking about something very technical that you think is just a matter of the frontend dev team, you might not know the whole team and some  from QA or backend knows about the framework you are working on. DO NOT underestimate anyone on the team.
+
+##Adopting one of this practices is a team work
+Before adopting one of the tips written here in this article on your team or organization, you might want to discuss it with the rest of your team. After that, it will be a process to fully adopt it, some colleagues  can forget about starting a thread, or to use a Yes emoji reaction instead of writing the answer. In those cases you can use the :start_a_thread: emoji, or :use_emojis: , or :thats_a_paddlin: to help your partners to get used to it.
```

Comment:
```suggestion
Before adopting one of the tips written in this article on your team or organization, you might want to discuss it with the rest of your team. After that, it will be a process to fully adopt it: some colleagues will forget to start a thread or to use a 'Yes' emoji reaction instead of writing the answer. In those cases you can use the :start_a_thread: emoji, or :use_emojis: , or :thats_a_paddlin: to help your partners to get used to it.
```